### PR TITLE
Explain purpose of the setupBastion method

### PR DESCRIPTION
**Pull request by jkaniuk**
Let's improve readability of gke.go

### CODE REVIEW

File: kubetest/gke.go [hunk 1]
```diff
@@ -608,14 +610,17 @@ func (g *gkeDeployer) setupBastion() error {
 	filtersToTry = append(filtersToTry, exactFilter)
 	// As a fallback - use proxy instance name as a regex but check only cluster nodes
 	var igFilters []string
+	// filter out VMs not belonging to the GKE cluster
```
Comment (tosi3k) [inline kubetest/gke.go:613]: Nitpick - please start sentences in the comments with the capital letters for consistency's sake.

**Comment by aojea** [comment]
/approve

Thanks

**Pull request merged**